vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,54 +1,97 @@
 ---
 id: 4RFQMNA91cRW12VJl_PFd
 title: Taking Notes
-desc: Taking Notes
-updated: 1636527192313
+desc: Creating notes, understanding hierarchy, and using Lookup to quickly find your notes
+updated: 1639427284878
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, use `Ctrl+L` / `Cmd+L` to bring up [[Dendron's Lookup|dendron://dendron.dendron-site/dendron.topic.lookup]]. This is a shortcut for the `Dendron: Lookup` command.
```

Comment:
can we combine the first two sentences? 

To create a note, run `Dendron: Lookup` (CTRL+L/CMD+L) to bring up the [[Lookup Bar|...]]